ddtrace/opentracer/span.py

```diff
@@ -49,14 +51,13 @@ def __init__(self, tracer, context, operation_name):
 
         super(Span, self).__init__(tracer, context)
 
+        self.log = SpanLog()
```

Comment:
I prefer to not implement something that we can't support right now. Otherwise we just keep things in memory that we'll never use.